sql/core/src/test/scala/org/apache/spark/sql/CollationSuite.scala

```diff
@@ -1625,6 +1626,63 @@ class CollationSuite extends DatasourceV2SQLBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("Expressions not supporting CS_AI collators") {
+    val unsupportedExpressions: Seq[Any] = Seq(
+      "ltrim",
+      "rtrim",
+      "trim",
+      "startswith",
+      "endswith",
+      "locate",
+      "instr",
+      "str_to_map",
+      "contains",
+      "replace",
+      ("translate", "efg"),
+      ("split_part", "2"),
+      ("substring_index", "2"))
+
+    val unsupportedCollator = "unicode_ai"
```

Comment:
```suggestion
    val unsupportedCollation = "unicode_ai"
```

Reply:
it's not a **collator** per se, but rather a collation (or, if you want - more precisely, a `collationName`)